app.js

```diff
@@ -1,4 +1,58 @@
 //  in this file create an express application - use the middle-ware built into express
+var express = require('express');
+var trucks = require('./trucks');
+var idManager = require('./idManager');
+
+idManager.setIds(trucks.getTrucks());
+
+var bodyParser = require('body-parser');
+var app = express();
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+var serveStatic = express.static('public');
+app.use(serveStatic);
+
+app.get('/trucks', function (request, response){
+        var trucklist = trucks.getTrucks();
+        response.json(trucklist);
+});
+
+app.get('/trucks/:id', function (request, response) {
+        var id = request.params.id;
+  
+        var truck = trucks.getTruck(id);
```

Comment:
Since the existing truck data we currently have doesn't have any id's, this won't return anything except for any new trucks you create and add to the foodTrucks array.